CHANGELOG.md

```diff
@@ -114,10 +120,10 @@ it will be removed; but as it won't be user-visible this isn't considered a brea
 - Security: Upgrading to the latest release of Golang as part of our general dependency upgrade
   process. This includes security fixes for CVE-2022-41725, CVE-2022-41723.
 
-- Feature: In Envoy 1.24, experimental support for a native OpenTelemetry tracing driver  was
-  introduced that allows exporting spans in the otlp format. Many  Observability platforms accept
+- Feature: In Envoy 1.24, experimental support for a native OpenTelemetry tracing driver was
```

Comment:
I'm not sure why these other formatting changes were made when I generate the Changelog, I'd be happy to remove though.

Reply:
Don't worry too much about it, usually it's just editor settings or something like that.